container_test/run_tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Script to run the python id_mapping_service_container_test.py locally or on GitHub Actions.
+# Builds and mounts id mapping service and mongo docker containers, and then calls
+# the python script.
+#
+# See .github/workflows/test.yml for GitHub Actions implementation.
+
+# build and start the containers
+docker compose up -d --build
+compose_up_exit_code=$?
+if [ $compose_up_exit_code -ne 0 ]; then
+    echo "Error: docker-compose up -d --build command failed with exit code $compose_up_exit_code."
+    exit $compose_up_exit_code
+fi
+
+export USER=myname
+# create user myname with token
+export TOKEN=$(docker exec id_mapper_container /bin/sh ./id_mapper --user $USER --create)
```

Comment:
This is a little confusing since this isn't really the token, it's the output of the create user command which is then parsed in the test script. I'd either rename it to something more accurate or change the contents so it actually is the token. Maybe something like this would work:

```
$ echo "$ID_MAPPER_OUTPUT"
some junk
token_here
$ echo "$ID_MAPPER_OUTPUT" | sed -n '2p'
token_here
```

Just adding the pipe and sed command to the end of this id mapper command line might work

Reply:
👍

Reply:
:+1: